### DISCUSSION TITLE
GPT4免费使用教程

### DISCUSSION THREAD

**Discussion by renqabs** [viopsa233/AI-testers-Forum#86] [技术交流]
~~本教程纯属搬运，教程的截图里有插件链接，安装后不要登录，直接点跳过，教程里的API key也是可以用的，这个API key来自教程里URL地址里提供的Discord获取，有需要的话可以加入Discord获取一下，(据说是黑客得到API之后免费分享的，大家可以看看新闻)，另外教程的作者是真的很无私，专门建了一个免费使用GPT-4的网站（是真的完全免费，没有套路），B站也有教程~~
~~API key: PQZ2wORga4eWMcnElEv2bVnlRtdYzmAU6ZIJgqzzUEY~~
~~URL: https://chimeragpt.adventblocks.cc~~
~~[Sider-GPT4配置教程.zip](https://github.com/ZS520L/GPT4-MidJourney-API/blob/main/Sider-GPT4%E9%85%8D%E7%BD%AE%E6%95%99%E7%A8%8B.zip)~~
~~https://www.bilibili.com/video/BV1ys4y167w9/?spm_id_from=333.999.0.0~~
~~https://www.bilibili.com/video/BV1SP411C7TQ/?spm_id_from=333.999.0.0~~
~~https://www.bilibili.com/video/BV1dh4y177uY/?spm_id_from=333.999.0.0~~
~~新的GPT4免费API [free-ai-apis](https://github.com/NovaOSS/free-ai-apis)~~
~~试验了下[NovaAI](https://nova-oss.com/)可以用GPT4，用法跟chimeraGPT差不多，大家也可以试试别的~~
新的免费GPT4 [gpt4-unlimited-phin](https://github.com/Lomusire/gpt4-unlimited-phin)，缺点是需要魔法

**Comment by NostalgiaGit** [comment]
感谢无私分享！试了下，API和Url确实可用，群里也可以获得新的api。用了几个测试题，确实是GPT4，虽然有个别回答不正确，可能是gpt4版本的问题。

**Reply by NostalgiaGit** [reply]
顺便，这个sider在我这用不了，选模型那没有gpt4的选项，选gpt3.5会出错。不过我用之前装的另一个extension叫chathub可以成功调用这个api和url，以及直接在url网页里用也可以。

**Reply by renqabs** [reply]
我这没问题啊，点击刷新试试，我用的edge浏览器
![image](https://github.com/viopsa233/AI-testers-Forum/assets/130155002/57a519af-bafb-4458-b73a-22fe07ef76e4)

**Reply by NostalgiaGit** [reply]
我这确实不行，edge dev浏览器，刷新也没有gpt4，很奇怪。不过不打紧，我在另一个extension里也能调用这个api，很好用！

**Reply by renqabs** [reply]
ChatGPT-Next-Web也挺好用的，直接下载编译好的就能用，配置是一样的

**Reply by NostalgiaGit** [reply]
多谢，这个不错，有桌面端，更方便。以后就用这个了:smiley:

**Reply by oureveryday** [reply]
链接设成： `https://chimeragpt.adventblocks.cc` ，而不是`https://chimeragpt.adventblocks.cc/`

**Reply by NostalgiaGit** [reply]
多谢，还真是那个斜杠的事，神奇😂

**Comment by twsec-pro** [comment]
![image](https://github.com/viopsa233/AI-testers-Forum/assets/58679784/defd66e9-c2e8-4c76-a8d9-8c389fc7aa35)
获取api的群咋加入？

**Reply by renqabs** [reply]
访问这个网址，点击Discord Link，好像需要魔法，最好用google浏览器
![image](https://github.com/viopsa233/AI-testers-Forum/assets/130155002/a5b9e07a-ab6e-40e7-bd3f-eb658d84cd6b)

**Reply by twsec-pro** [reply]
![image](https://github.com/viopsa233/AI-testers-Forum/assets/58679784/0b038afa-a983-457c-b6e0-3b6377e3468c)
进去之后在哪个栏可以获取呢？

**Reply by NostalgiaGit** [reply]
在任意聊天频道发/key get就可以了，看上去那个ai chatroom应该就行（我进去的群不是这样的，频道设置不一样）

**Reply by twsec-pro** [reply]
能拉下你进的群吗？或者分享下链接

**Reply by NostalgiaGit** [reply]
我就是从楼主给的那个网页里进的群，[邀请链接](https://discord.gg/t9QvFvzkmV)，进去后会有一条私信告诉你获得key的方法，就是上面说的在任意聊天频道发/key get就行，比如在#chat频道发就可以

**Comment by 15849769265** [comment]
![Uploading image.png…]()
请问为啥我没有权限发送消息呢

**Reply by renqabs** [reply]
先验证一下才会有权限发消息，点击左侧的verify进行验证

**Reply by 15849769265** [reply]
谢谢，成功了

**Comment by jishux2** [comment]
![屏幕截图(9)](https://github.com/viopsa233/AI-testers-Forum/assets/120390643/934ec2d2-568c-421e-a49c-b10e0a8dd470)
怎么一直提示更新失败呢，已经换过三四个api了。但是[https://chimeragpt.adventblocks.cc/](https://chimeragpt.adventblocks.cc/) 里面的chimeragpt又是可以用这个密钥的

**Reply by renqabs** [reply]
使能“使用我自己的网址”，填入https://chimeragpt.adventblocks.cc/
![image](https://github.com/viopsa233/AI-testers-Forum/assets/130155002/c02167a2-218a-4b47-a790-01594fb4f419)

![image](https://github.com/viopsa233/AI-testers-Forum/assets/130155002/415be102-6c5c-4934-92f1-5020d6f08267)

**Reply by jishux2** [reply]
这样啊😂感谢感谢

**Comment by synge0718** [comment]
有没有手机版yandex浏览器能用的chatgpt扩展，sider在yandex上会提示不兼容无法安装

**Comment by ghost** [comment]
对比0314版GPT-4，0613版无比垃圾，而Bing是最强大的，所以有Bing没必要用这个

**Reply by ghost** [reply]
是32k的bing不好用吗，去用这个不稳定的8k

**Reply by NostalgiaGit** [reply]
new bing的逻辑能力有时不如chatgpt4，比如下面这道逻辑题
> A man with a boat needs to cross the river. He can only take one thing at a time besides himself. He needs to take the cabbage, a goat and a lion across the river. The cabbage cannot be left alone with the goat. The goat cannot be left alone with the lion and the lion cannot be left alone with the cabbage. How will the man solve this puzzle and get all three things across the river?

这是个经典题但被人改了题干，添加了狮子不能和菜在一起的新规则，这个条件下该题无解。这个只有chatgpt4能准确判断出无解，其他包括new bing都不行。还有一个pi和3.2谁大的问题new bing也不能回答正确但chatgpt4可以。不过偶尔也有new bing表现更好的时候，但似乎比较受限于服务器状态。

但不知道写代码的能力new bing和这个0613版的gpt4谁好，有人有体验吗？

**Reply by ghost** [reply]
官网GPT-4：
This is a classic puzzle, known as the "river crossing" problem. In this version, we have a lion, a goat, and a cabbage. Let's solve this.

1. First, the man takes the goat across the river, because it's the most problematic item (it can't be left alone with either of the other two). Now, on the other side of the river, we have the man and the goat, and on the starting side, we have the lion and the cabbage.

2. The man leaves the goat on the other side and goes back to the starting side to get the cabbage. He takes the cabbage across the river. Now, on the other side of the river, we have the man and the cabbage, and on the starting side, we have the lion. But we have left the goat on the other side.

3. To solve this, the man leaves the cabbage on the other side, takes the goat back to the starting side. Now, on the other side of the river, we have the cabbage, and on the starting side, we have the man, the goat, and the lion.

4. The man leaves the goat on the starting side and takes the lion across the river. Now, on the other side of the river, we have the man and the lion, and the cabbage, and on the starting side, we have the goat.

5. Finally, the man leaves the lion on the other side and goes back to the starting side to get the goat. He takes the goat across the river. Now, on the other side of the river, we have the man, the lion, the goat, and the cabbage.

This way, the man has managed to bring all three items across the river without any of them being eaten by another.

Bing：
![Bing creative mode](https://github.com/viopsa233/AI-testers-Forum/assets/138886027/8ce7c6de-d7fc-4a6c-ad3b-2069f5881088)

![Bing precise mode](https://github.com/viopsa233/AI-testers-Forum/assets/138886027/2edaec61-6d7a-457f-a0bf-ece8d7d718bd)

**Reply by ghost** [reply]
![chrome_screenshot](https://github.com/viopsa233/AI-testers-Forum/assets/138886027/1dc6b4b1-f2af-415f-bbe9-a56bedb5d26e)

**Reply by ghost** [reply]
结论：半斤八两
用英文问官网GPT-4给出的答案是 Pi is bigger. The approximate value of pi is 3.14159, which is slightly less than 3.2. 第一句错第二句对

**Reply by trz98** [reply]
![Screenshot_2023-07-30-11-00-48-451_ai.sider.ChatGPT.android.jpg](https://github.com/viopsa233/AI-testers-Forum/assets/111866065/6ee9a96b-0f03-47be-8a09-5c6334b3c25a)

第一次用的是gbt4，第二次用的是gbt3.5，用的是chimeragbt接口。

**Reply by ghost** [reply]
![Precise](https://github.com/viopsa233/AI-testers-Forum/assets/138886027/ecacfdec-0b13-4daa-93f5-eeb0479c7dab)
创造模式不行

**Reply by NostalgiaGit** [reply]
我用楼里提供的这个Chimera API的gpt4模型是可以正确回答这两个问题的：

逻辑题的回答1：指出题干可能有错误，修正后回答
<details><summary>展开折叠内容</summary>
![puzzle1](https://github.com/viopsa233/AI-testers-Forum/assets/132838751/54680203-c7c6-4d3c-8f64-f983163312d9)
</details>

逻辑题的回答2：再次提问，回答一样，换了种表达方式，还是说题干有问题，修正条件后回答
<details><summary>展开折叠内容</summary>
![puzzle1-2](https://github.com/viopsa233/AI-testers-Forum/assets/132838751/eaf5a65d-2963-449b-884c-39129648ec7c)
</details>
之前我提问时还有明确回答无解的。

最后pi的问题：简单明了，回答正确
<details><summary>展开折叠内容</summary>
![puzzle2](https://github.com/viopsa233/AI-testers-Forum/assets/132838751/8ee6bdaa-34ff-4826-8a26-1402557b827f)
</details>

**Reply by NostalgiaGit** [reply]
哎，我的回复折叠后截图无法正确显示了，不太熟悉github的回复格式，本来添加折叠代码是为了避免占用太多版面空间的。我直接把其中一段回答文字粘贴过来吧，

This sounds like a lot of fun! However, there seems to be a contradiction in the rules. It says:

1. The cabbage cannot be left alone with the goat, which means the goat will eat the cabbage.
2. The goat cannot be left alone with the lion, which means the lion will eat the goat.
3. And, the lion cannot be left alone with the cabbage, which implies the lion will eat the cabbage.

However, if the lion eats both the goat and the cabbage, then there's no situation in which every item could be left alone with the lion, as that would result in their disappearance. There has to be at least one animal that wouldn't be threatened by one other item in order for the puzzle to be logically solvable.

For the sake of progression, I'm going to adjust the rules a bit, making the lion the top predator that will eat the goat, the goat would then eat the cabbage but the cabbage wouldn't do harm to any. So:

1. The goat cannot be left alone with the lion (since the lion will eat the goat).
2. The cabbage cannot be left alone with the goat (since the goat will eat the cabbage).
3. The lion poses no threat to the cabbage.

Here's a way to solve it:
后面的解题步骤就省略了，总之我这Chimera调用的gpt4模型对这个问题回答挺好的，问了多次都是这样

**Comment by synge0718** [comment]
提示：提供的这个key已经被ChimeraGPT方面封禁，使用该key访问时会提示：“This API key has been suspended for violating ChimeraGPT's policies.”建议加入他们的discord服务器获得一个自己的key后再来使用
![1690190309783](https://github.com/viopsa233/AI-testers-Forum/assets/31887102/55939282-b148-41a1-ba34-54e9de40a145)

**Reply by renqabs** [reply]
现在想加入discord也很困难了，得用tun模式还有discord客户端

**Reply by Devotee4658** [reply]
Discord获取key的群一直加不进去，应该是限制了吧

**Reply by trz98** [reply]
验证的时候把代理关掉刷新，然后再打开代理可以通过vpn检测验证。

**Reply by Devotee4658** [reply]
老哥详细说下，我用的网页版，我点那个验证的时候关了代理，进网站之后通过验证然后开代理回到discord网页还是显示没通过

**Comment by oureveryday** [comment]
最新的地址需要填这个
`https://chimeragpt.adventblocks.cc/api`

**Reply by NostalgiaGit** [reply]
大佬nb！之前我这好几个app都用不了了，换成你这个地址果然又能用了。但是奇怪的是我看群里的说明似乎是让用另一个地址：
> https://chimeragpt.adventblocks.cc/api/v1/chat/completions

我找了半天最后找到一个叫better ChatGPT的app可以成功使用上述地址调用api。但奇怪的是better ChatGPT只能用上面这个地址，用你这个地址会失败，而我之前用的app（比如上面推荐的sider和chatgpt next web）又只能用你给的这个地址，用群里那个以completions结尾的也会无效。搞不懂到底是什么原理。

**Reply by synge0718** [reply]
我用安卓版的sider试了一下，确实用https://chimeragpt.adventblocks.cc/api/v1/chat/completions这个地址了，sider插件应该也差不多

**Reply by renqabs** [reply]
这个是专门用于chimeraGPT的，可以直接在web里使用chimeraGPT的key（支持联网，作者会同步discord通知调整代码，更新也很及时），有兴趣的话可以去部署一个，https://huggingface.co/spaces/monra/freegpt-webui-chimera

**Reply by synge0718** [reply]
有点怪，安卓版的sider用的是`https://chimeragpt.adventblocks.cc/api/v1/chat/completions`，但是sider插件版只能用`https://chimeragpt.adventblocks.cc/api`

**Comment by synge0718** [comment]
经过实测，安卓版sider需要使用`https://chimeragpt.adventblocks.cc/api/v1/chat/completions`，但是sider插件以及ChatGPTBox都要使用`https://chimeragpt.adventblocks.cc/api`

**Reply by NostalgiaGit** [reply]
突然发现隔壁的SydneyQt也能用这个api，在设置的endpoint里要填这个地址：
> https://chimeragpt.adventblocks.cc/api/v1

就是必须以v1结尾，然后模型那里手动填gpt-4，居然也能用，神奇。但这个地址在其他app里不能用，只能填你列的这两个地址之一。

**Comment by xuyufenfei** [comment]
为什么我的频道和你们不一样
![Screenshot_2023-07-29-21-21-45-643_com discord](https://github.com/viopsa233/AI-testers-Forum/assets/133896005/5be4a79d-9264-4858-a5db-4e59ce5db09c)

**Reply by xuyufenfei** [reply]
来了，和大伙说一下惨痛经历，第一次验证没经验，vpn开太早了，导致多次验证失败，然后让把我号锁了，不让我验证，然后我就去注册另一个账号，没想到这个验证居然把我有另一个账号检测出来了，然后我就试了很多方法，用不同设备，买号，虚拟机，结果全检测出来了，验证没通过，花了30多买虚拟手机号

**Reply by NostalgiaGit** [reply]
现在验证这么困难吗，我加的早，倒是没遇到这些问题。不行试试其他的，我发现 https://github.com/ading2210/poe-api#documentation 这里也有一个poe逆向的项目，不过没用过不知道体验如何。

**Reply by Devotee4658** [reply]
老哥详细说下，我用的网页版，我点那个验证的时候关了代理，进网站之后通过验证然后开代理回到discord网页还是显示没通过

**Reply by xuyufenfei** [reply]
说明你的账号创建未满两天

**Reply by Devotee4658** [reply]
这样啊，感谢老哥。那我注册个号等两天看看。验证次数失败过多那个号是不是废了

**Reply by xuyufenfei** [reply]
如果这样的话，你可以去那个验证平台的频道去open ticket，我就是这样，第二天就解封了

**Reply by Devotee4658** [reply]
好的，我试试

**Comment by iwannamore** [comment]
@renqabs 
1.B站那个up写的那个本地端是有积分限制的,没看懂那是什么,能解释一下吗?
2.discord的那个个人key接口有限制吗?在哪里可以查看?
作者写的指引我没看明白,整半天才弄到key.

**Reply by iwannamore** [reply]
刚刚在chat频道获得的key,测试出来得到的模型是gpt-3

**Reply by ghost** [reply]
别再问为什么是GPT-3这个问题了，这问题都不知道被问了几次了，如果不信是GPT-4就掏钱用官网吧

**Reply by renqabs** [reply]
没有限制，能拿到key就可以用，可以不用up主的本地端，ChatGPT Next Web（web或者客户端）、ChatGPT Sidebar（插件）、freegpt-webui chimeragpt-version（web）都是可以用这个key的，把端点设置对了就可以

**Reply by iwannamore** [reply]
刚刚在freegpt-webui chimeragpt-version（web）测试了一下,奇怪的是虽然我提问问它是什么模型,他说自己是gpt-3,但是再问几下(我选项选的是gpt-4)就提示我gpt-4 has rate limit.这可能证实了给的key是4以及有速率限制这两个实施,所以我才这么问

**Reply by renqabs** [reply]
是有速率限制的，discord公告应该有

**Reply by iwannamore** [reply]
看到了2/min,我就说

**Reply by renqabs** [reply]
![image](https://github.com/viopsa233/AI-testers-Forum/assets/130155002/c5a46b12-5f9f-4636-a426-f46bd16603b9)

**Reply by iwannamore** [reply]
我已经找到这个网页了谢谢,话说能问问你平常都怎么应用吗,如果不能拿来用的话,食之无味啊.

**Reply by renqabs** [reply]
自己部署一个freegpt-webui chimeragpt-version，render上就可以免费部署，一般用GPT-4写代码，联网搜索以及娱乐使用bing ai

**Reply by iwannamore** [reply]
好的,谢谢

**Reply by xuyufenfei** [reply]
大佬教教端点怎么设置，我是next web的项目

**Reply by xuyufenfei** [reply]
api已经拿到了，但直接填入没有用

**Reply by tzcnbo** [reply]
兄弟你是不是没填chimeragpt的链接啊 https://chimeragpt.adventblocks.cc/api
![image](https://github.com/viopsa233/AI-testers-Forum/assets/105526202/fe14dd17-1c6b-4c6b-a674-64d28f955eec)

**Comment by tzcnbo** [comment]
GPT4用不了了，只有3.5turbo可以用

**Comment by ghost** [comment]
因为现在gpt-4不能用了

**Reply by xuyufenfei** [reply]
我3.5也不能用

**Reply by xuyufenfei** [reply]
搞好了，可以用4，只是有限制，每分钟两条好像

**Comment by xuyufenfei** [comment]
关于4不能使用的问题，群里有人说是限制了中文，用英文正常使用

**Reply by GraceGemini** [reply]
我还是觉得有Sydney不用这个

**Reply by renqabs** [reply]
现在好像又好一些了，不过GPT-4时不时的就会抽风，用起来确实挺费劲的

**Reply by Devotee4658** [reply]
还真是，一用中文八成以上就抽风，切换成英文就一点问题没有😂

**Comment by synge0718** [comment]
我号也被chimeragpt封了，但是我用的也不多呀

**Reply by renqabs** [reply]
每隔2三天就得验证一次，不然就封

**Comment by Joackk** [comment]
现在似乎进不去了啊，官网的邀请链接也被提示过期了，有遇到一样情况的吗？

**Reply by tcxl000** [reply]
邀请一直提示过期

**Reply by renqabs** [reply]
据说是群没了。。。

**Reply by xuyufenfei** [reply]
群没了，但是之前搞得秘钥还能用

**Reply by Joackk** [reply]
key还能用？我的key就是挂了去群看才发现群没了

**Reply by renqabs** [reply]
使用Nova AI吧，每天赠送2500额度，可累计，GPT-4每次消耗30额度

**Reply by Joackk** [reply]
有免费的3.5  api吗？

**Reply by renqabs** [reply]
有的，每次消耗10个额度

**Reply by Joackk** [reply]
这个额度赠送是怎么算的？进了频道每天就自动送还是要操作一下？

**Reply by renqabs** [reply]
每天自动送，最好隔几天去看一下，免得长时间不活动被踢

**Reply by Joackk** [reply]
我看那个赠送介绍时，10级以上会送2.5，担纲加进去都是1级，会送吗？

**Reply by renqabs** [reply]
0级每天2500

**Reply by Joackk** [reply]
soga，感谢大佬解惑

**Comment by Devotee4658** [comment]
请教下楼主老哥，Nova AI的discord群加进去了，API也拿到了，但是我看其他人command频道中有等级显示，但我这一直没有，机器人一直提示 “You have no rank. Keep chatting to earn a rank!”请问这个该怎么弄？

**Reply by NostalgiaGit** [reply]
去#general频道随便说两句话就好，只有这个频道说话有经验能升级。只能说英文，不能无意义灌水。随便接句话或者干脆问个好都可以，别不停刷屏就行。现在管得严了等级没以前好升了，不过也还好，0级的2500也够用了，而且这个积分系统说是要改，可能某天后大家就都清零重来了。

**Reply by Devotee4658** [reply]
Ok，多谢老哥解答

**Comment by Devotee4658** [comment]
今天Nova也用不了了
